recipes/spyder/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "2.3.9" %}
+
+package:
+  name: spyder
+  version: {{ version }}
+
+source:
+  fn:  spyder-{{ version }}.zip
+  url: https://pypi.io/packages/source/s/spyder/spyder-{{ version }}.zip
+  md5: dd01e07a77123c128ff79ba57b97c1d7
+
+build:
+  number: 0
+  osx_is_app: True
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - pyqt
+    - rope
+    - pyflakes
+    - jedi  >=0.8
+    - qtconsole
+    - nbconvert
+    - pygments
+    - sphinx
+    - pep8
+    - psutil
+    - python.app  # [osx]
+
+test:
+  commands:
+    - USER=test spyder -h  # [unix]
```

Comment:
It's fine. We should be doing a better job so that you don't have to do this. Just felt like I should add a note to explain to myself and others why we need to change the docker image to fix this.